pkg/kubelet/network/exec/exec.go

```diff
@@ -131,3 +159,39 @@ func (plugin *execNetworkPlugin) TearDownPod(namespace string, name string, id k
 	glog.V(5).Infof("TearDownPod 'exec' network plugin output: %s, %v", string(out), err)
 	return err
 }
+
+func (plugin *execNetworkPlugin) Status(namespace string, name string, id kubeletTypes.DockerID) (*network.PodNetworkStatus, error) {
+	out, err := utilexec.New().Command(plugin.getExecutable(), statusCmd, namespace, name, string(id)).CombinedOutput()
+	glog.V(5).Infof("Status 'exec' network plugin output: %s, %v", string(out), err)
+	if err != nil {
+		return nil, err
+	}
+	if string(out) == "" {
+		return nil, nil
+	}
+	findVersion := struct {
+		api.TypeMeta `json:",inline"`
+	}{}
+	err = json.Unmarshal(out, &findVersion)
```

Comment:
This is OK for now, but we should really unmarshal into a TypeMeta, check the version and kind, and then unmarshal into a versioned object.  We can do that when we have a second version :)